src/zos_console/zowe/zos_console_for_zowe_sdk/console.py

```diff
@@ -40,17 +42,17 @@ def issue_command(self, command: str, console: Optional[str] = None) -> dict:
 
         Returns
         -------
-        dict
+        IssueCommandResponse
             A JSON containing the response from the console command
         """
         custom_args = self._create_custom_request_arguments()
         custom_args["url"] = self._request_endpoint.replace("defcn", console or "defcn")
         request_body = {"cmd": command}
         custom_args["json"] = request_body
         response_json = self.request_handler.perform_request("PUT", custom_args)
-        return response_json
+        return IssueCommandResponse(response_json)
 
-    def get_response(self, response_key: str, console: Optional[str] = None) -> dict:
+    def get_response(self, response_key: str, console: Optional[str] = None) -> GetReponseResponse:
```

Comment:
`GetReponseResponse` -> `GetResponseResponse`

If this sounds too redundant, we could change it to a more generic name like `TsoDataResponse` 😋